code/modules/mob/living/autohiss.dm

```diff
@@ -35,6 +28,7 @@
 			to_chat(src, "Auto-hiss is now OFF.")
 
 /datum/species
+	var/has_autohiss = FALSE // set in global_list.dm, line 158
```

Comment:
Do we really need comments like this? It'll quickly become incorrect - And people have find-in-files.